docs/reference/ml/df-analytics/apis/get-inference-trained-model-metadata.asciidoc

```diff
@@ -0,0 +1,736 @@
+[role="xpack"]
+[testenv="basic"]
+[[get-inference-metadata]]
+= Get {infer} trained model metadata API
+[subs="attributes"]
+++++
+<titleabbrev>Get {infer} trained model metadata</titleabbrev>
+++++
+
+Retrieves training metadata information for trained {infer} models.
+
+experimental[]
+
+
+[[ml-get-inference-metadata-request]]
+== {api-request-title}
+
+`GET _ml/inference/_metadata` +
+
+`GET _ml/inference/_all/_metadata` +
+
+`GET _ml/inference/<model_id>/_metadata` +
+
+`GET _ml/inference/<model_id>,<model_id_2>/_metadata` +
+
+`GET _ml/inference/<model_id_pattern*>,<model_id_2>/_metadata`
+
+
+[[ml-get-inference-metadata-prereq]]
+== {api-prereq-title}
+
+If the {es} {security-features} are enabled, you must have the following
+privileges:
+
+* cluster: `monitor_ml`
+
+For more information, see <<security-privileges>> and {ml-docs-setup-privileges}.
+
+[[ml-get-inference-metadata-desc]]
+== {api-description-title}
+
+You can get training metadata information for multiple trained models in a single API
+request by using a comma-separated list of model IDs or a wildcard expression.
+
+
+[[ml-get-inference-metadata-path-params]]
+== {api-path-parms-title}
+
+`<model_id>`::
+(Optional, string)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
+
+
+[[ml-get-inference-metadata-query-params]]
+== {api-query-parms-title}
+
+`allow_no_match`::
+(Optional, boolean)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=allow-no-match]
```

Comment:
> allow_no_match ...

I don't think you can use this shared definition, since it's referring to jobs, not models. I notice the same problem in other APIs, so I'll create that missing definition in a separate PR.

Reply:
See the new "allow-no-match-models" in https://github.com/elastic/elasticsearch/pull/62008

Reply:
```suggestion
include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=allow-no-match-models]
```